datumaro/components/dataset.py

```diff
@@ -1249,7 +1250,11 @@ def import_from(
             if eager:
                 dataset.init_cache()
         except _ImportFail as e:
-            raise e.__cause__
+            cause = e.__cause__ if getattr(e, "__cause__", None) is not None else e
```

Comment:
I recommend to use `hasattr`

Reply:
Thanks for the comment but `hasattr` would return `True` always since `__cause__` attribute is set to another `Exception` if the exception is raised from it, otherwise `None`.